### DISCUSSION TITLE
Customizable dialogs

### DISCUSSION THREAD

**Discussion by elicul** [xamarin/XamarinCommunityToolkit#1085] [Ideas]
Hi guys what do you think of adding customizable dialogs to Xamarin Community Toolkit? Currently, I'm using the dialogs from [XF-Material-Library](https://github.com/Baseflow/XF-Material-Library) repository but they aren't being updated and I need to use the whole library just for the dialogs. I know that Xamarin.Forms have dialogs but they are really hard to style.

**Comment by pictos** [comment]
@elicul I believe that you can do it with our PopUp control. Did you try it?

**Reply by elicul** [reply]
@pictos Yes I know I wanted to use PopUps to create them. It would be used just to enable faster developments.

**Reply by pictos** [reply]
@elicul, so are you asking to add predefined styles for those popUps?
